microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,186 @@
+## Chromium color themes have been deprecated
+
+<!-- Subtitle: Users of Chromium themes in DevTools will be automatically migrated to the default dark or light theme for a more reliable experience. -->
+
+To improve reliability, the **Chromium Light** and **Chromium Dark** themes have been deprecated.  Users who previously had these themes selected will be automatically migrated to the **Light+** or **Dark+** theme, respectively:
+
+DevTools in **Light+** theme: 
+
+![DevTools in Light+ theme](./devtools-106-images/light-plus-theme.png)
+
+DevTools in **Dark+** theme:
+
+![DevTools in Dark+ theme](./devtools-106-images/dark-plus-theme.png)
+
+There are many DevTools color themes to choose from, such as **Monokai** and **Solarized**.  To change the color theme in DevTools, click the **Settings** (![Settings icon](../../../media/settings-gear-icon-light-theme.png)) button, and then in the **Preferences** page, in the **Theme** dropdown menu, select a theme:
+
+![Setting the DevTools theme to Monokai](./devtools-106-images/set-theme-monokai.png)
+
+See also:
+* [Apply a color theme to DevTools](../../../customize/theme.md)
+
+
+<!-- ====================================================================== -->
+## Accessibility improvements in Focus Mode and high contrast mode
+
+<!-- Subtitle: The new Dock location and Activity Bar location buttons in Focus Mode now work better with screen readers, and computed styles are easier to see in high contrast mode. -->
+
+
+#### Dock location and Activity Bar location buttons in Focus Mode now work better with screen readers
+
+In Microsoft Edge 105, Focus Mode received several improvements, including new **Dock location** and **Activity Bar location** buttons.  In Microsoft Edge 106, these new buttons now work better with assistive technologies, such as screen readers.
+
+Screen readers now announce which **Dock location** button or which **Activity Bar location** button is currently selected, and the number of buttons, representing the available location options:
+
+![The redesigned Docking menu in Focus Mode](./devtools-106-images/focus-mode-redesigned-docking-menu.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+* [Navigate DevTools with assistive technology](../../../accessibility/navigation.md)
+* [Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View](../../../whats-new/2022/09/devtools-105.md#focus-mode-improved-location-controls-for-devtools-activity-bar-and-quick-view) in _What's new in DevTools (Microsoft Edge 105)_.
+
+
+#### Computed styles are easier to see in high contrast mode
+
+In high contrast mode, in previous versions of Microsoft Edge, the expand and collapse buttons for computed styles didn't render correctly, in the **Computed** section of the **Styles** pane in the **Elements** tool.  In Microsoft Edge 106, this issue has been fixed.
```

Comment:
I don't understand this sentence. There is no Computed section in the Styles pane. The Elements tool has a sidebar where you can use either the Styles pane, or the Computed pane (among other things). But the Styles pane itself doesn't contain a Computed section.

Reply:
changed to "in the **Computed** tab in the **Elements** tool"

![image](https://user-images.githubusercontent.com/45407486/192623524-ed51629c-34c6-4c74-93f7-ee507e79c45d.png)